static/app/views/performance/onboarding.tsx

```diff
@@ -101,6 +107,54 @@ function Onboarding({organization}: Props) {
       duration,
     });
   }
+  const showSampleTransactionBtn = organization.features.includes(
+    'performance-create-sample-transaction'
+  );
+  const featureTourBtn = (
+    <FeatureTourModal
+      steps={PERFORMANCE_TOUR_STEPS}
+      onAdvance={handleAdvance}
+      onCloseModal={handleClose}
+      doneUrl={performanceSetupUrl}
+      doneText={t('Start Setup')}
+    >
+      {({showModal}) => (
+        <Button
+          priority={showSampleTransactionBtn ? 'link' : 'default'}
+          onClick={() => {
+            trackAnalyticsEvent({
+              eventKey: 'performance_views.tour.start',
+              eventName: 'Performance Views: Tour Start',
+              organization_id: parseInt(organization.id, 10),
+            });
+            showModal();
+          }}
+        >
+          {t('Take a Tour')}
+        </Button>
+      )}
+    </FeatureTourModal>
+  );
+  const secondaryBtn = showSampleTransactionBtn ? (
+    <Button
+      onClick={async () => {
+        trackAdvancedAnalyticsEvent(
+          'growth.performance_sample_transaction',
```

Comment:
@Neo-Zhixing let's make this a `performance_views` event to fit in with the other events on this page.